.github/workflows/main.yaml

```diff
@@ -31,18 +31,9 @@ jobs:
         echo "HPE_DEPLOY_REPO=$(echo ${{ github.repository }} | tr '[:upper:]' '[:lower:]')" >> $GITHUB_ENV
 
     - name: set TAG & TESTTAG for main/master
-      if: ${{ github.event.repository.default_branch == 'main' || github.event.repository.default_branch == 'master' }}
       run: |
-        echo "TAG=${{ github.sha }}" >> ${GITHUB_ENV}
+        echo "TAG=$(./git-version-gen)" >> ${GITHUB_ENV}
```

Comment:
This doesn't show me the value that git-version-gen returned.
I think you still want to tag with github.sha as well.

Reply:
`cat`'ed the github env file, and pushed up the SHA tag as well.